earth_enterprise/src/fusion/rasterfuse/TmeshPackgenTraverser.cpp

```diff
@@ -150,6 +157,13 @@ TmeshPackgenTraverser::TmeshPackgenTraverser(
       writer_(file_pool_, output, true /* overwrite */),
       extra_cache_(20000 /* apx 160M */),
       not_covered_tiles_exist_(false) {
+
+  if((decimation_threshold <= 0.0) || (config.decimate == false)) {
```

Comment:
minor: float comparison. can you pass a very small value (ex. 0.000000001) and verify that comparison. likely we may not have to worry about very small numbers.

Reply:
+1

Reply:
Are you suggesting that 0.0 is not the same as 0.000000000 ?

Reply:
I guess my ex did not apply to float size, but the idea stands, though it is minor. comparing float or double is not an exact thing like integer. Usually it is done with comparing the diff of the abs of the values with some very small value (epsilon) to know for sure the values are less/greater/equal. I think here it is not an issue unless if we expect a very tiny decimation value :)

Reply:
I'm not sure that issue applies to 0.0. It's pretty much zero or not. Since this value doesn't come from any sort of calculation, but from direct user input, I don't think we have to worry about rounding error. If someone types 0.00000000000001 into the command line, then I would assume they meant for it to be non-zero and should trigger the decimation code.

Reply:
Yes it was very minor. We can disregard this.